src/main/java/org/jabref/gui/Base.css

```diff
@@ -98,6 +98,10 @@
     -jr-drag-target: -jr-purple;
     -jr-drag-target-hover: derive(-jr-purple, 80%);
 
+    /* For summary text color */
```

Comment:
This comment can be removed (as it is nearly verbatim the name of the CSS class)